scripts/blogView.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
 // create object for all of the functions that control what blog articles you see
-var blogview = {};
+const blogview = {};
 
 // Populate filters with the authors and categories from the blogsource
-blogview.populateFilters = function() {
-  $('article').not('.template').each(function() {
+blogview.populateFilters = function () {
+  $('article').not('.template').each(function () {
```

Comment:
When you implemented handlebars, didn't you remove the article template (with the class of 'template')? If so, what does ".not('.template')" do?